oci/private/tarball.sh.tpl

```diff
@@ -27,8 +27,23 @@ for LAYER in $(${YQ} ".[]" <<< $LAYERS); do
     cp "${IMAGE_DIR}/blobs/${LAYER}" "${BLOBS_DIR}/${LAYER}.tar.gz"
 done
 
+# format the repotags as a json array:
```

Comment:
I think you could do this inside the `yq` filter below and it would be a lot simpler? allow the `env(repotags)` to continue to read the value as it's formatted now, and use some `yq` function to convert to an array?

Reply:
Hmm so I tried:
```
repotags="$REPOTAGS" \
layers="${LAYERS}" \
"${YQ}" eval \
        --null-input '.[0] = {"Config": env(config), "RepoTags": env(repotags) | split(" "), "Layers": env(layers) | map( "blobs/" + . + ".tar.gz") }' \
        --output-format json > "${TARBALL_MANIFEST_PATH}"
```

And that worked for a single repotag but when I added multiple I got this error:
```
Error: Value for env variable 'config' not provided in env()
```

The for-loop worked though.